docs/source/RunningInEmulationTCG.rst

```diff
@@ -0,0 +1,92 @@
+.. _emulation:
+
+===============================
+Running in emulation mode (TCG)
+===============================
+
+Sometimes it's useful to run Avocado-vt in emulation mode (eg. when
+checking PR about different architecture, or to debug certain feature
+while executing the test). As this is not default, extra arguments are
+usually required.
+
+Shared for all architectures is that you need the image. We supply
+JeOS for most of the architectures, so you should be able to get
+it via::
+
+    $ avocado vt-bootstrap --vt-guest-os JeOS.$VERSION.$ARCH
+
+where:
+
+* ``VERSION`` is JeOS version (when writing this document it was ``27``)
+* ``ARCH`` is the desired architecture (eg. aarch64)
+
+Or you can run any of the ``unattended_install`` tests with
+``--vt-guest-os`` of your choice (very slow).
+
+When running the tests, on top of the usual arguments, you usually
+need to specify:
+
+* ``--vt-qemu-bin`` - path to binary that supports expected architecture
+  (eg. ``/usr/local/bin/qemu-system-aarch64``)
+* ``--vt-qemu-dst-bin`` - for migration tests you need also to specify
+  the destination qemu binary (otherwise default one is used)
+* ``enable_kvm=no`` - to disable KVM (if necessary)
+
+
+aarch64
+=======
+
+ARM always requires `cpu_model` as well as `machine_type`. To
+get list of available models you can run ``qemu-system-aarch64
+-cpu help -M virt`` (note: not all listed cpus are bootable).
```

Comment:
Does it help to list the CPU models that work? Since jeos is a 64bit OS, only cortex-a53, cortex-a57, cortex-a72, and max work (the support for cortex-a72 and max are added to upstream version last year. Not sure if they are available in RHEL or Fedora). I guess 'max' should work with qemu-system-aarch64 on x86_64 host, but I didn't verify it.

Reply:
> Well these models are evolving fairly quickly (compare to how frequently we are to touch this documentation). There is the example command and I assume people will usually start with that. I don't think it's necessary to add it here.

OK

> I was not aware of "max", it works well on Fedora. Hmm, do you think I should list that instead of the cortex-a57?

I prefer to keep cortex-a57 because it's a real cpu model name and easy to understand.